packages/stores/src/queries/pool-share/index.ts

```diff
@@ -244,11 +244,11 @@ export class ObservableQueryPoolShare {
 
   fetch(bech32Address: string) {
     return Promise.all([
-      this.queryPools.fetch(),
```

Comment:
Fetch will only fetch if stale, and is not async. `waitFreshResponse` will fetch regardless if it is stale, basically guaranteeing new data. Fetch should be rarely used, since the query store will automatically keep data up to date.